test/hierarchy/README.md

```diff
@@ -25,3 +25,7 @@ ee, _ := CA.IssuePrecertificate(...)
 cert, _ := issuance.LoadCertificate("test/hierarchy/int-e1.cert.pem")
 test.AssertEqual(t, issuance.GetIssuerNameID(ee), issuer.NameID())
 ```
+
+## Files
+* `multiple-roots.cert.pem` is a concatenation of `root-x1.cert.pem` and `root-x2.cert.pem`
+* `duplicate-roots.cert.pem` is `root-x2.cert.pem` concatenated twice
```

Comment:
Above this, under `# Boulder Test Hierarchy`, we state:

> This directory does not include private keys for the roots, as Boulder should never perform any operations which require access to root private keys.

```suggestion
## Files
* `multiple-roots.cert.pem` is a concatenation of `root-x1.cert.pem` and `root-x2.cert.pem`
* `duplicate-roots.cert.pem` is `root-x2.cert.pem` concatenated twice
```

We'll need to update that to avoid confusion. Probably the best way to clarify here, is to explain that we're using these for gRPC in remoteva. It doesn't have to be detailed.

Reply:
Disregard this, I misread your additions.